sql/moz-fx-data-shared-prod/firefox_ios_derived/attributable_clients_v1/query.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
+RETURNS INT64 AS (
+  (SELECT SUM(value) FROM UNNEST(map))
+);
+
+WITH client_search_activity AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    sample_id,
+    client_info.client_id,
+    SUM(sum_map_values(metrics.labeled_counter.search_in_content)) AS searches,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_with_ads)) AS searches_with_ads,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_ad_clicks)) AS ad_clicks,
+  FROM
+    firefox_ios.baseline
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+  GROUP BY
+    submission_date,
+    sample_id,
+    client_id
+),
+adjust_client_info AS (
+  SELECT
+    client_id,
+    sample_id,
+    first_seen_date,
+    adjust_network,
+    adjust_ad_group AS adjust_adgroup,
+    adjust_campaign,
+    adjust_creative,
+    metadata.is_reported_first_session_ping,
+  FROM
+    firefox_ios.firefox_ios_clients
+  WHERE
+    adjust_network <> "Unknown"
```

Comment:
We should also disinclude "Other", which is accounting for a large proportion of our installs now. Can you check what that is in the underlying data? ([source](https://sql.telemetry.mozilla.org/queries/91936#227584))

Reply:
This appears to be mostly `Organic` and `Untrusted Devices` ([graph](https://sql.telemetry.mozilla.org/queries/92097/source#227890)). I wonder if we should also pass through `Organic` here @fbertsch ?